genTestingItems.js

```diff
@@ -0,0 +1,12 @@
+const ITEM_LIST = [];
+
+for (i = 0; i < 20; i++) {
+    ITEM_LIST.push({
+        name: 'Bag of holding',
+        type: 'Armor',
+        durability: Math.floor(Math.random() * 100) + 1,
+        enhancement: Math.floor(Math.random() * 16) + 1,
```

Comment:
Instead of having the `+1` here, you could use `Math.ceil` instead:
```javascript
durability: Math.ceil(Math.random() * 100),
enhancement: Math.ceil(Math.random() * 16),
```

You could also make the argument that a helper function could be made for making random numbers:
```javascript
const randNum = num => Math.ceil(Math.random() * num);

for (let i = 0; i < 20; i++) {
    ITEM_LIST.push({
        name: 'Bag of holding',
        type: 'Armor',
        durability: randNum(100),
        enhancement: randNum(16),
    });
}
```